mantidproject/mantid
# GroupDetectors v2 TimeIndex support

GroupDetectors v2 will now preserver time indexes correctly when merging.

**To test:**

Make use of the changes in PR #19398. Use `CreateSampleWorkspace` to create a workspace with time indexes. Try out different methods of grouping together detectors, and check time indexes are set correctly.

Fixes #19679.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
